src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementEventService.java

```diff
@@ -240,6 +248,116 @@ public EventSummary updateProcurementEvent(final Integer procId, final String ev
 
   }
 
+  /**
+   * Get all suppliers on an event.
+   *
+   * @param procId
+   * @param eventId
+   * @return
+   */
+  public Collection<OrganizationReference> getSuppliers(final Integer procId,
+      final String eventId) {
+
+    log.debug("Get suppliers for event '{}'", eventId);
+
+    var event = validationService.validateProjectAndEventIds(procId, eventId);
+    var existingRfx = jaggaerService.getRfx(event.getExternalEventId());
+    var orgs = new ArrayList<OrganizationReference>();
+
+    if (existingRfx.getSuppliersList().getSupplier() != null) {
+      orgs = (ArrayList<OrganizationReference>) existingRfx.getSuppliersList().getSupplier()
```

Comment:
Completely optional, but if you wanted to avoid the reassignment to `orgs` you could use `.forEachOrdered(orgs::add)` instead of the collector at the end of the steam (negating the need for the cast too) 👍